rails/level-t/app/controllers/links_controller.rb

```diff
@@ -0,0 +1,72 @@
+class LinksController < ApplicationController
+  before_action :set_link, only: %i[ show edit update destroy ]
+
+  # GET /links or /links.json
+  def index
+    @links = Link.all
```

Comment:
Hi, 
Here we can use SQL to limit the database search to only 6 items.
Currently, all rows are fetched from the database and then filtered in Ruby

[Cf documentation](https://apidock.com/rails/ActiveRecord/FinderMethods/take) for futher information
```suggestion
    @links = Link.take(6)
```